.gitignore

```diff
@@ -14,3 +14,6 @@ glide.lock
 .DS_Store
 .terraform/
 gpg/
+
+# ignore. For testing we don't want to store a specific provider.
+.terraform.lock.hcl
```

Comment:
Should this still be ignored?

Reply:
Yes, if we keep this then whenever the provider is changed (since locally one can download the binary) then the lock file will keep an old version cached and throw errors. To prevent this, we should ignore the lock file